src/Core/src/Handlers/Page/PageHandler.iOS.cs

```diff
@@ -25,7 +25,15 @@ public static void MapBackground(IPageHandler handler, IContentView page)
 		{
 			if (handler is IPlatformViewHandler invh && invh.ViewController is not null)
 			{
-				invh.ViewController.View?.UpdateBackground(page);
+				if (page.Background is ImageSourcePaint image)
```

Comment:
Why is this logic in the handler, rather than the extension method?

Reply:
Created new extensions methods.